apps/vmq_server/priv/vmq_server.schema

```diff
@@ -277,6 +277,13 @@
                                                                           hidden
                                                                          ]}.
 
+%% @doc specifies the connect timeout.
+{mapping, "connect_timeout", "vmq_server.connect_timeout", [
```

Comment:
isn't `socket_connect_timeout` more descriptive?

Reply:
hmm, maybe sth like `mqtt_connect_timeout` would make it clear that it refers to MQTT connections?

Reply:
Yes this is actually better since it is explicit and there are multiple listeners and very generic can confuse users